src/controllers/CheckYourAppealController.ts

```diff
@@ -43,13 +44,19 @@ export class CheckYourAppealController extends SafeNavigationBaseController<any>
             .map(info => info[SignInInfoKeys.UserProfile])
             .unsafeCoerce();
 
-        return {
+        const model = {
             ...super.prepareViewModelFromSession(session),
             userProfile
         };
+        loggerInstance()
+            .debug(`${CheckYourAppealController.name} - prepareViewModelFromSession: ${JSON.stringify(model)}`);
+
+        return model;
     }
 
     protected prepareViewModelFromAppeal(appeal: Appeal): any {
+        loggerInstance()
```

Comment:
I find this useless especially that `prepareViewModelFromSession` logs prepared version of view model and this function does nothing. Basically both logs will be roughly same.